modules/v2x/fusion/libs/common/base/v2x_object_supplement.h

```diff
@@ -0,0 +1,231 @@
+/******************************************************************************
```

Comment:
It seems there are many duplicates in modules/perception/base/object_supplement.h. So do some cleanup.